packages/cli/src/service/pipeline.ts

```diff
@@ -3,6 +3,7 @@ import * as Url from 'url';
 import { PipelineResp, PluginStatusValue, PipelineConfig } from '@pipcook/sdk';
 import { constants, PluginStatus } from '@pipcook/pipcook-core';
 import { readJson } from 'fs-extra';
+import { prompt } from 'inquirer';
```

Comment:
I think it's not good behavior to write cli logic in service.

Reply:
Updated.